.github/workflows/release.yml

```diff
@@ -3,53 +3,85 @@ name: Build Release
 on: 
   workflow_dispatch:
 
-env:
-  packageName: "com.vrchat.demo-template"
+jobs:
+
+  # Validate Repository Configuration
+  config:
+    runs-on: ubuntu-latest
+    outputs:
+      config_package: ${{ steps.config_package.outputs.configPackage }}
+    steps:
 
-permissions:
-  contents: write
+    # Ensure that required repository variable has been created for the Package
+    - name: Validate Package Config
+      id: config_package
+      run: |
+        if [ "${{ vars.PACKAGE_NAME }}" != "" ]; then
+          echo "configPackage=true" >> $GITHUB_OUTPUT;
+        else
+          echo "configPackage=false" >> $GITHUB_OUTPUT;
+        fi
 
-jobs:
+  # Build and release the Package
+  # If the repository is not configured properly, this job will be skipped
   build:
+    needs: config
     runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    env:
+      packagePath: Packages/${{ vars.PACKAGE_NAME }}
+    if: needs.config.outputs.config_package == 'true'
     steps:
-    
+
+      # Checkout Local Repository
       - name: Checkout
-        uses: actions/checkout@v3
-    
-      - name: get version
+        uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac
+
+      # Get the Package version based on the package.json file
+      - name: Get Version
         id: version
-        uses: notiz-dev/github-action-json-property@7c8cf5cc36eb85d8d287a8086a39dac59628eb31
+        uses: zoexx/github-action-json-file-properties@b9f36ce6ee6fe2680cd3c32b2c62e22eade7e590
```

Comment:
Just curious - why do you like `zoexx/github-action-json-file-properties` over `notiz-dev/github-action-json-property` for this task?

Reply:
That change was actually made in #11, I left it to avoid unnecessary conflicts with their PR. Perhaps that change resolved one of the warnings that was previously being generated?

Reply:
That change was actually made in #11, I left it to avoid unnecessary conflicts with their PR. Perhaps that change resolved one of the warnings that was previously being generated?

Reply:
Haha - indeed it was, you even left a comment about it over there.